modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -237,7 +237,7 @@ Database aliases can be created using `CREATE ALIAS`.
 The command supports the creation of both local and remote database aliases.
 For more information on local and remote database aliases as part of a composite database, see xref::database-administration/composite-databases/manage-composite-databases.adoc#create-composite-database-alias[Create database aliases in composite databases].
 
-The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Access control -> DBMS administration].
```